confidant/settings.py

```diff
@@ -379,6 +379,22 @@ def str_env(var_name, default=''):
     'WEBHOOK_PASSWORD',
     str_env('WEBHOOK_PASSWORD')
 )
+# Webhook Array File configuration
+# Example:
+# cat webhook_file.json
+# [
+#   {
+#     "WEBHOOK_URL": "https://webhook_dev.example.com"
+#   },
+#   {
+#     "WEBHOOK_URL": "https://webhook_test.example.com",
+#     "WEBHOOK_USERNAME": "myhookuser",
+#     "WEBHOOK_PASSWORD": "mylongandsupersecurehookpassword"
```

Comment:
passwords should really be part of the encrypted settings, otherwise we're storing secrets in the clear somewhere.

Reply:
Hey @ryan-lane thanks for your feedback.

For my clarity, I have a few linked questions:

1. Is it still OK to have the `WEBHOOK_ARRAY_FILE` store the value for `WEBHOOK_PASSWORD` in plain text? I'd like to rely on the container management engine to place the file onto the file system accessible by the confidant container.
2. If (1) is OK, I'm feeling there will be changes to the `settings.py` and `encrypted_settings.py` files to loop through the file and then possibly store the encrypted passwords in a dict.

Reply:
The encrypted_settings code is there to load a set of values from a pre-encrypted SECRETS_BOOTSTRAP environment variable. It's there to avoid needing to load any plain text passwords into confidant. I'd prefer to not have confidant load any secrets from plaintext.